server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerScanMetrics.java

```diff
@@ -49,7 +49,7 @@ public void addYield(long value) {
 
   public void incrementOpenFiles(int delta) {
     Math.max(0, delta);
```

Comment:
Without assignment, this statement is unnecessary.  Sorry - I didn't make it a multi-edit suggestion.

Reply:
No, you're right, good catch. I've made like 4 mistakes on this simple change. I think I just need to start my vacation now.

Reply:
Or - more coffee